src/System.Private.CoreLib/shared/System/Globalization/CompareInfo.Unix.cs

```diff
@@ -15,7 +15,7 @@ namespace System.Globalization
     public partial class CompareInfo
     {
         [NonSerialized]
-        private Interop.Globalization.SafeSortHandle _sortHandle = null!; // initialized in helper called by ctors
+        private Interop.Globalization.CriticalSortHandle _sortHandle = null!; // initialized in helper called by ctors
```

Comment:
Please add a comment that this must not be explicitly Dispose'd.  If it ever needs to be, this will need to change back to being a SafeHandle; otherwise, if it could end up being Dispose'd concurrently with P/Invokes, and we could end up in bad situations.

Reply:
(I assume we don't explicitly Dispose it because we expect that the vast majority of CompareInfos will be cached / singletons?)